packages/dapp/src/components/Dashboard/ActivityListItem.tsx

```diff
@@ -3,10 +3,10 @@ import { connect, DispatchProp } from "react-redux";
 import { Link } from "react-router-dom";
 import { formatRoute } from "react-router-named-routes";
 import BigNumber from "bignumber.js";
-import { BoostForm } from "@joincivil/civil-sdk";
+import styled from "styled-components";
```

Comment:
`BoostForm` import removed here (not sure how it'd build/lint actually because it's used below).